packages/webamp-modern-2/src/utils.ts

```diff
@@ -80,6 +80,20 @@ export function findLast<T>(
   }
 }
 
+
+export function getUrlQuery(location:Location, variable:string):string {
```

Comment:
Is it possible to run Prettier?

Reply:
how? I have installed the Prettier in my vs-code.

Reply:
oh! I found it. just run Format Document, right?

Reply:
Yeah, that should work. I moved Prettier up to the project root, (root of the yarn workspace) which might cause the plugin to not work correctly.

At the very least you can do: 

```
yarn prettier -w ./packages/webamp-modern-2/src/utils.ts
```